src/utils_request.js

```diff
@@ -0,0 +1,272 @@
+import rqst from 'request';
+import contentType from 'content-type';
+import util from 'util';
+import zlib from 'zlib';
+import * as url from 'url';
+import { decompressStream } from 'iltorb';
+
+const FIREFOX_MOBILE_USER_AGENT = 'Mozilla/5.0 (Android 9.0; Mobile; rv:66.0) Gecko/66.0 Firefox/66.0';
```

Comment:
We should finalize the PR for the UserAgents package and use it here.

Reply:
yep, I will do the PR